lib/JsonpMainTemplatePlugin.js

```diff
@@ -38,6 +38,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"script.charset = 'utf-8';",
 			"script.async = true;",
 			"script.timeout = " + chunkLoadTimeout + ";",
+			"var cached = script.readyState && script.readyState === 'loaded';",
```

Comment:
state of the script in this position wouldn't be loaded, it is always uncomplete here. So fix will not help